config/values.yaml

```diff
@@ -0,0 +1,27 @@
+serviceAccount:
+  # Annotations to add to the service account (like the ARN of the IRSA role)
+  annotations: {}
+
+controller:
+  defaultImage: ko://github.com/awslabs/karpenter/cmd/controller
+# fill the variable below if you want to override the default image
+#  image:
+#  tag:
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity:
```

Comment:
Karpenter builds multiple images for different architectures, which are automatically detected from the image manifest. I think we can get away without these node affinities.

Reply:
Agreed